papi/app.py

```diff
@@ -49,16 +62,37 @@ def get_cards(deck_id):
 
         return jsonify(decks)
 
+    else:
+        invalid = {'id': 'Error: not a valid deck_id'}
+        return jsonify( invalid )
+
 @app.route('/api/v1/decks/<deck_id>/balances', methods=['GET'])
 def balances(deck_id):
-    balances = []
-    Balances = db.session.query(Balance).all()
-
-    for balance in Balances:
-        balance = balance.__dict__
-        del balance['_sa_instance_state']
-        balances.append(balance)
-        print(balance)
+
+    length_check = len(str(deck_id))
+    if length_check == 64:
+        try:
+            int(deck_id, 16) #if not hex will cause error, failing test
+            germ_free = True
+        except:
+            germ_free = False
+    else:
+        germ_free = False
```

Comment:
Let's make a function out of germ_free so we can reuse it. We can put it inside data.py and have it return Boolean.